### PR TITLE
creating of consents collection

### PR DESCRIPTION
**Pull request by SamirTouira**
Related to : #4327 
First step : creation of collection "consents" with interface 
Draw.io has been updated

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -12,13 +12,8 @@
         "lint": {
```
Comment (phpgeekfr) [inline angular.json:12]: can you revert formatting changes so only your changes are visible?

File: libs/consents/src/lib/consents.module.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  imports: [CommonModule],
+})
+export class ConsentsModule {}
```
Comment (phpgeekfr) [inline libs/consents/src/lib/consents.module.ts:7]: this file is not required, remove it

File: libs/consents/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  "include": [],
+  "references": [
+    {
+      "path": "./tsconfig.lib.json"
+    },
+    {
+      "path": "./tsconfig.spec.json"
+    }
+  ]
+}
```
Comment (phpgeekfr) [inline libs/consents/tsconfig.json:13]: update this file to look like other libraries

File: libs/consents/src/index.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export * from './lib/consents.module';
```
Comment (phpgeekfr) [inline libs/consents/src/index.ts:1]: this file is not required right now

File: nx.json [hunk 5]
```diff
@@ -9,16 +9,10 @@
   },
```
Comment (phpgeekfr) [inline nx.json:9]: same as `angular.json` file

File: tsconfig.base.json [hunk 6]
```diff
@@ -59,7 +59,8 @@
       "@blockframes/media/*": ["libs/media/src/lib/*"],
       "@blockframes/testing/*": ["libs/testing/src/lib/*"],
       "@blockframes/firebase-utils": ["libs/firebase-utils/src/index.ts"],
-      "@blockframes/campaign/*": ["libs/organization/campaign/src/lib/*"]
+      "@blockframes/campaign/*": ["libs/organization/campaign/src/lib/*"],
+      "@blockframes/consents": ["libs/consents/src/index.ts"]
```
Comment (phpgeekfr) [inline tsconfig.base.json:63]: since `index.ts` is not needed right now, can you update to reflect other libs config ?

File: tsconfig.json [hunk 7]
```diff
@@ -1,3 +0,0 @@
-{
-  "extends": "./tsconfig.base.json"
-}
```
Comment (phpgeekfr) [inline tsconfig.json:3]: do not remove this one

**Review by phpgeekfr — CHANGES_REQUESTED** [review]

File: tsconfig.base.json (hunk 6 above)
Comment (SamirTouira) [reply tsconfig.base.json:63]: the only file in blockframes is this right ? if I've understood, I can delete the line 63 and delete aswell index.ts if it's not needed for the moment

Comment (phpgeekfr) [reply tsconfig.base.json:63]: for me, `""@blockframes/consents": ["libs/consents/src/index.ts"]` should look like `"@blockframes/consents/*": ["libs/consents/src/lib/*"],`  and yes index.ts can be deleted

File: tsconfig.json (hunk 7 above)
Comment (SamirTouira) [reply tsconfig.json:3]: from my side, I've got `  "extends": "../../tsconfig.base.json"` in line 2, isn't the right way ?

Comment (phpgeekfr) [reply tsconfig.json:3]: In this PR you are removing this file `tsconfig.json`. Don't remove it

**Review by phpgeekfr — APPROVED** [review]

**Pull request merged**